### PR TITLE
systemd: update to 250.3 and include Jinja2

### PR DESCRIPTION
**Pull request by heitbaum**
new build dependency used by systemd v249 

systemd >= v249 requires Jinja2 for templating
- refer: https://github.com/systemd/systemd/pull/19630: Use jinja2 for templating
- m4 use is dropped, and the two parallel substitution systems in meson are unified.

can be added at anytime. 
Not required until systemd v249
Including Jinja2 with the v250 systemd

Systemd NEWS: https://github.com/systemd/systemd/blob/main/NEWS

systemd: update from v247 to v250

issues: 
- FIXED: need to track down error: `systemd-journald[360]: Failed to write entry (22 items, 760 bytes), rotating before retrying: Argument list too long`
- FIXED: why is my docker console output getting into `journalctl -f`
- FIXED: Issue with 250-rc3 - systemd/systemd 21829 - temp patch -
c7f6f36 - fixed in https://github.com/systemd/systemd/pull/21846

### CODE REVIEW

File: packages/python/devel/Jinja2/package.mk [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
```
Comment (CvH) [inline packages/python/devel/Jinja2/package.mk:1]:
```suggestion
# SPDX-License-Identifier: GPL-2.0
```

File: packages/python/devel/Jinja2/package.mk [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+# Copyright (C) 2021-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="Jinja2"
+PKG_VERSION="3.0.1"
+PKG_SHA256="703f484b47a6af502e743c9122595cc812b0271f661722403114f71a79d0f5a4"
+PKG_LICENSE="GPL"
```
Comment (CvH) [inline packages/python/devel/Jinja2/package.mk:7]:
```suggestion
PKG_LICENSE="BSD"
```
image

File: packages/python/devel/Jinja2/package.mk [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+PKG_SITE="https://pypi.org/project/Jinja2/"
+PKG_URL="https://files.pythonhosted.org/packages/source/${PKG_NAME:0:1}/${PKG_NAME}/${PKG_NAME}-${PKG_VERSION}.tar.gz"
+PKG_DEPENDS_HOST="Python3:host setuptools:host MarkupSafe:host"
+PKG_LONGDESC="Jinja is a fast, expressive, extensible templating engine. Special placeholders in the template allow writing code similar to Python syntax. Then the template is passed data to render the final document."
```
Comment (CvH) [inline packages/python/devel/Jinja2/package.mk:11]:
```suggestion
PKG_LONGDESC="Jinja is a fast, expressive, extensible templating engine."
```

**Comment by CvH** [comment]
merge blocked till systemd bump

**Comment by heitbaum** [comment]
Changed to WIP. Have included v250-rc2 systemd. Will remove WIP once systemd v250 is released.

**Comment by heitbaum** [comment]
Issue with 250-rc3 - https://github.com/systemd/systemd/pull/21829 - temp patch - 
c7f6f36

**Comment by heitbaum** [comment]
Tested correctly now. GtG from my testing.
```
nuc11:~ # dmesg | grep systemd
[    2.715460] systemd[1]: systemd 250.1 running in system mode (-PAM -AUDIT -SELINUX -APPARMOR -IMA -SMACK -SECCOMP -GCRYPT -GNUTLS -OPENSSL -ACL +BLKID -CURL -ELFUTILS -FIDO2 +IDN2 -IDN -IPTC +KMOD -LIBCRYPTSETUP -LIBFDISK -PCRE2 -PWQUALITY -P11KIT -QRENCODE -BZIP2 -LZ4 -XZ -ZLIB -ZSTD -BPF_FRAMEWORK -XKBCOMMON +UTMP +SYSVINIT default-hierarchy=hybrid)
[    2.715507] systemd[1]: Detected architecture x86-64.
[    2.792935] systemd[1]: Queued start job for default target Kodi Mediacenter Interface.
[    2.795370] systemd[1]: Created slice Slice /system/modprobe.
[    2.795456] systemd[1]: Created slice User and Session Slice.
[    2.795475] systemd[1]: Started Dispatch Password Requests to Console Directory Watch.
[    2.795489] systemd[1]: Started Forward Password Requests to Wall Directory Watch.
[    2.795504] systemd[1]: Reached target Path Units.
[    2.795512] systemd[1]: Reached target Slice Units.
[    2.796981] systemd[1]: Listening on RPCbind Server Activation Socket.
[    2.796998] systemd[1]: Reached target RPC Port Mapper.
[    2.797026] systemd[1]: Listening on initctl Compatibility Named Pipe.
[    2.797421] systemd[1]: Journal Audit Socket was skipped because of a failed condition check (ConditionSecurity=audit).
[    2.797520] systemd[1]: Listening on Journal Socket (/dev/log).
[    2.797558] systemd[1]: Listening on Journal Socket.
[    2.797739] systemd[1]: Listening on udev Control Socket.
[    2.797769] systemd[1]: Listening on udev Kernel Socket.
[    2.797801] systemd[1]: Huge Pages File System was skipped because of a failed condition check (ConditionPathExists=/sys/kernel/mm/hugepages).
[    2.798011] systemd[1]: Mounting POSIX Message Queue File System...
[    2.798233] systemd[1]: Mounting Kernel Debug File System...
[    2.798488] systemd[1]: Mounting Kernel Trace File System...
[    2.798799] systemd[1]: Mounting Variable Directory...
[    2.799118] systemd[1]: Starting Create List of Static Device Nodes...
[    2.799407] systemd[1]: Starting Load Kernel Module configfs...
[    2.799690] systemd[1]: Starting Load Kernel Module drm...
[    2.799950] systemd[1]: Starting Load Kernel Module fuse...
[    2.800285] systemd[1]: Starting Show Version...
[    2.800551] systemd[1]: Starting Create Persistent Log Directory on /storage and rotate large logs...
[    2.800801] systemd[1]: Starting Mounting swapfile...
[    2.803248] systemd[1]: Starting Rebuild Hardware Database...
[    2.805053] systemd[1]: Starting Load Kernel Modules...
[    2.805327] systemd[1]: Starting Setup NTP servers for timesyncd...
[    2.805603] systemd[1]: Starting Coldplug All udev Devices...
[    2.805897] systemd[1]: Starting Wait for Kernel Time Synchronisation...
[    2.806689] systemd[1]: Mounted POSIX Message Queue File System.
[    2.806726] systemd[1]: Mounted Kernel Debug File System.
[    2.806750] systemd[1]: Mounted Kernel Trace File System.
[    2.806773] systemd[1]: Mounted Variable Directory.
[    2.806862] systemd[1]: Finished Create List of Static Device Nodes.
[    2.806940] systemd[1]: modprobe@configfs.service: Deactivated successfully.
[    2.807022] systemd[1]: Finished Load Kernel Module configfs.
[    2.807094] systemd[1]: modprobe@drm.service: Deactivated successfully.
[    2.807148] systemd[1]: Finished Load Kernel Module drm.
[    2.807222] systemd[1]: modprobe@fuse.service: Deactivated successfully.
[    2.807279] systemd[1]: Finished Load Kernel Module fuse.
[    2.807362] systemd[1]: Finished Show Version.
[    2.807893] systemd[1]: Mounting FUSE Control File System...
[    2.808226] systemd[1]: Mounting Kernel Configuration File System...
[    2.808642] systemd[1]: Finished Create Persistent Log Directory on /storage and rotate large logs.
[    2.809462] systemd[1]: Persistent Log Storage was skipped because all trigger condition checks failed.
[    2.809917] systemd[1]: Finished Setup NTP servers for timesyncd.
[    2.812312] systemd[1]: Mounted FUSE Control File System.
[    2.812367] systemd[1]: Mounted Kernel Configuration File System.
[    2.816915] systemd[1]: Finished Load Kernel Modules.
[    2.817472] systemd[1]: Starting Apply Kernel Variables...
[    2.824552] systemd[1]: Finished Apply Kernel Variables.
[    2.844578] systemd[1]: Finished Coldplug All udev Devices.
[    2.853512] systemd[1]: Finished Mounting swapfile.
[    2.853541] systemd[1]: Reached target Swaps.
[    2.853840] systemd[1]: Mounting Temporary Directory /tmp...
[    2.854704] systemd[1]: Mounted Temporary Directory /tmp.
[    2.854742] systemd[1]: Reached target Local File Systems.
[    2.855135] systemd[1]: Starting Setup machine-id...
[    2.869941] systemd[1]: Finished Setup machine-id.
[    2.870756] systemd[1]: Starting Journal Service...
[    2.871138] systemd[1]: Starting Create Static Device Nodes in /dev...
[    2.874268] systemd[1]: Finished Create Static Device Nodes in /dev.
[    2.884275] systemd-journald[361]: Data hash table of /run/log/journal/98626b9930184724b0e20606f52cf421/system.journal has a fill level at 75.1 (1537 of 2047 items, 524288 file size, 341 bytes per hash table item), suggesting rotation.
[    2.884277] systemd-journald[361]: /run/log/journal/98626b9930184724b0e20606f52cf421/system.journal: Journal header limits reached or header out-of-date, rotating.
[    2.885305] systemd[1]: Started Journal Service.
[    2.888606] systemd-journald[361]: Received client request to flush runtime journal.
```

**Review by CvH — APPROVED** [review]

**Pull request merged**